.github/scripts/main.sh

```diff
@@ -87,8 +87,8 @@ export OCAMLRUNPARAM=b
     test -d _opam || opam switch create . --empty
     eval $(opam env)
     opam pin --kind=path $GITHUB_WORKSPACE --yes --no-action
-    opam pin . -yn
-    opam install opam-rt --deps-only
+    opam pin . -yn --ignore-pin-depends
```

Comment:
It is already tested on current branch

Reply:
The issue is the `--empty` 3 lines above... if ocaml-system is selected (which is available from the cache) then this proceeds